webapp-django/crashstats/base/views.py

```diff
@@ -3,6 +3,16 @@
 from django.shortcuts import render
 
 
+class HttpResponseBadRequest(http.HttpResponseBadRequest):
+    """Override of the Django HttpResponseBadRequest that makes sure
+    to set the content_type to "text/plain" if it hasn't already been
+    set."""
+    def __init__(self, *args, **kwargs):
+        if 'content_type' not in kwargs:
+            kwargs['content_type'] = 'text/plain; charset=UTF-8'
```

Comment:
There are just too many places for that. And all the places where we have just a short `return HttpResponseBadRequest(str(form.errors))` would have to be quite a lot expanded to have to write `...., content_type='text/plain; charset=UTF-8')`

I'm also wondering, should perhaps